File: src/main/java/com/DavidM1A2/AfraidOfTheDark/common/block/mangrove/BlockMangrovePlanks.java
/*
 * Author: David Slovikosky
 * Mod: Afraid of the Dark
 * Ideas and Textures: Michael Albertson
 */
package com.DavidM1A2.AfraidOfTheDark.common.block.mangrove;

import com.DavidM1A2.AfraidOfTheDark.common.block.core.AOTDBlock;

import net.minecraft.block.SoundType;
import net.minecraft.block.material.Material;

public class BlockMangrovePlanks extends AOTDBlock
{
	public BlockMangrovePlanks()
	{
		super(Material.WOOD);
		this.setUnlocalizedName("mangrove_planks");
		this.setRegistryName("mangrove_planks");
		this.setSoundType(SoundType.WOOD);
		this.blockHardness = 2.0F;
		this.blockResistance = 1.0F;
	}
}

File: src/main/java/com/davidm1a2/afraidofthedark/common/block/mangrove/BlockMangrovePlanks.kt
package com.davidm1a2.afraidofthedark.common.block.mangrove

import com.davidm1a2.afraidofthedark.common.block.core.AOTDBlock
import net.minecraft.block.SoundType
import net.minecraft.block.material.Material

/**
 * Class representing mangrove planks
 *
 * @constructor for mangrove planks sets the planks properties
 */
class BlockMangrovePlanks : AOTDBlock("mangrove_planks", Material.WOOD)
{
    init
    {
        setHardness(2.0f)
        this.soundType = SoundType.WOOD
    }
}